src/GraphBuilder.js

```diff
@@ -49,17 +49,15 @@ export default class GraphBuilder {
    * @param {rdf.Term} baseURI
    * @param {Object} value looks something like this:
    *   { 'resourceTemplate:bf2:Note':
-   *     { '67Bm64T0p2s': { 'http://www.w3.org/2000/01/rdf-schema#label': [Object] },
-   *       '555m64M0f3z': { 'http://www.w3.org/2000/01/rdf-schema#label': [Object] } } }
+   *     { 'http://www.w3.org/2000/01/rdf-schema#label': [Object],
+   *       'http://www.w3.org/2000/01/rdf-schema#label': [Object] } }
    */
   buildTriplesForNestedObject(baseURI, value) {
     // Is there ever more than one base node?
-    for (const key in value) {
-      Object.keys(value[key]).forEach((predicateListKey) => {
-        // we don't have a resourceURI to send as the second argument in the propertyTemplates yet.
-        this.buildTriplesForNode(baseURI, null, value[key][predicateListKey])
-      })
-    }
+    Object.keys(value).forEach((rtId) => {
```

Comment:
Small suggestion, you may be able to use `Object.values(value).forEach((rt) => {` and then on line 59, you could just pass in resourceTemplate directly as `rt`.